.github/compat-data-helper/.gitignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
I've installed the package locally (which is I guess how you would develop this bot further and run tests locally). Should we also add node_modules/ to the ignores here?

Reply:
Good idea! (I didn't catch this because I have `node_modules` in my global config.)